site/usage/webcomponent.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: usage
+menu: usage
+title: Embedding Vega-Lite with vega-element
+permalink: /usage/webcomponent.html
+---
+
+[Web components]((https://www.webcomponents.org/introduction) are a set of web platform APIs that allow you to create new custom, reusable, encapsulated HTML tags to use in web pages and web apps.
+
+[`vega-element`](https://www.webcomponents.org/element/PolymerVis/vega-element) is a Polymer webcomponent that can be used to embed vega or vega-lite visualization using custom HTML tags.
+
+**Quick start**
+The simplest usage case is to just embed a vega-lite visualization.
+```html
+<!-- import vega-element webcomponent from CDN (not for production use!!) -->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-element-cdn.html">
+<!-- import vega-tooltip (required only if using vega-tooltip plugin)-->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-tooltip.html">
+
+<!--
+hover flag enables hover event processing
+tooltip flag enables vega-tooltip plugin
+-->
+<vega-element hover tooltip vega-lite-spec-url="vl-spec.json"></vega-element>
+```
+
+Export SVG or PNG
+```html
+<!-- import vega-element webcomponent from CDN (not for production use!!) -->
+<link rel="import" href="https://rawgit.com/PolymerVis/vega-element/polymer2/vega-element-cdn.html">
+
+<!-- do not render anything when headless flag is set -->
+<vega-element id="chart" headless vega-lite-spec-url="vl-spec.json"></vega-element>
```

Comment:
What is headless for?

Reply:
It is more for nodejs which is not applicable for webcomponents. But a possible use case  is when you want to just want to export multiple charts without rendering. i.e. use the  element as an API.

Reply:
I think this needs to be explained.